mlflow/pyspark/ml/log_model_allowlist.txt

```diff
@@ -0,0 +1,44 @@
+# classification
+pyspark.ml.classification.LinearSVCModel
+pyspark.ml.classification.DecisionTreeClassificationModel
+pyspark.ml.classification.GBTClassificationModel
+pyspark.ml.classification.LogisticRegressionModel
+pyspark.ml.classification.RandomForestClassificationModel
+pyspark.ml.classification.NaiveBayesModel
+
+# clustering
+pyspark.ml.clustering.BisectingKMeansModel
+pyspark.ml.clustering.KMeansModel
+pyspark.ml.clustering.GaussianMixtureModel
+
+# Recommendation
+pyspark.ml.recommendation.ALSModel
```

Comment:
ALS shouldn't be in the allowlist right?
```suggestion
```

Reply:
Good catch !